src/src/pages/capabilities/MyCapabilities.js

```diff
@@ -210,11 +213,21 @@ export default function MyCapabilities() {
                 sx: {
                   borderRadius: '0',
                 }
-              }
-              }
-              enablePagination={false}
-              enableTopToolbar={false}
-              enableBottomToolbar={false}
+              }}
+              muiTopToolbarProps={{
+                sx: {
+                  background: 'none',
+                }
+                }}
+              muiBottomToolbarProps={{
+                sx: {
+                  background: 'none',
+                }
+              }}
+              enableDensityToggle={false}
+              enableHiding={false}
+              enableFilters={true}
```

Comment:
![image](https://github.com/dfds/selfservice-portal/assets/58768740/50c004da-a1bf-4710-b9c0-b6f3d7deaea6)

The pop up filters are not aligned (it is not nice for the eye). The last one (arrow) should not be shown.

Reply:
Cleaned up as per our real life conversation